packages/manager/apps/telecom/src/app/telecom/pack/move/translations/Messages_fr_FR.json

```diff
@@ -203,7 +204,8 @@
   "pack_move_confirm_no_commitment": "Ce changement d’offre n’est sujet à aucun réengagement.",
   "pack_move_confirm_commitment": "Un réengagement de {{ month }} mois débutera à la date de facturation.",
   "pack_move_confirm_new_offer_price": "Votre nouvel abonnement mensuel s'élève à : <span class=\"text-price\">{{ price }}</span> <span class=\"text-infos\">HT / mois</span>, applicable dès livraison de votre nouvel accès. Vous en serez informé par e-mail.",
-  "pack_move_confirm_shipping": "Vous avez sélectionné le mode de livraison de votre nouveau modem par transporteur (+ <span class=\"text-price\">{{ transportPrice }} €</span> <span class=\"text-infos\">HT</span>). Votre première mensualité sera donc de <span class=\"text-price\">{{ price }}</span> <span class=\"text-infos\">HT</span>.",
+  "pack_move_confirm_shipping": "Vous avez sélectionné le mode de livraison de votre nouveau modem par transporteur (+ {{ price }} € HT).",
```

Comment:
Same as here:
https://github.com/ovh/manager/pull/8294/files#r1012547118

Reply:
Hello, we sell internet access only in france.